flux/stdlib/influxdata/influxdb/schema/read_tag_keys_test.flux

```diff
@@ -0,0 +1,157 @@
+package universe_test
```

Comment:
did you see these added tests fail before your changes?

Reply:
My expectation is that `read_tag_keys` and `read_tag_values` would have passed before these changes, and the `_with_pred` variants would have failed. I'll double-check.

Reply:
Confirmed!